lib/sww.js

```diff
@@ -52,13 +53,28 @@ ServiceWorkerWare.prototype.handleEvent = function sww_handleEvent(evt) {
 
 ServiceWorkerWare.prototype.onFetch = function sww_onFetch(evt) {
   var steps = this.router.match(evt.request.method, evt.request.url);
+
+  // Push the fallback middleware at the end of the list if it's defined
+  // also being sure that wont be executed if there is a response already.
+  if (this.fallbackMw) {
+    var fallback = this.fallbackMw;
+    steps.push(function(req, res) {
```

Comment:
I think you are right, in terms of not forcing a specific behavior. Let hit the default one if we consider it necessary.

But so far, until we get https://bugzilla.mozilla.org/show_bug.cgi?id=1165860 I would like to keep it like that, otherwise all our request will be falling and going to fetch.

So my point when coding this was:
- Ok I need https://bugzilla.mozilla.org/show_bug.cgi?id=1165860 , so in order to not to fail now I'm going to hit the default just when no other mw was invoked.
- As soon as we have https://bugzilla.mozilla.org/show_bug.cgi?id=1165860 or even better, in the PR for that bug, that constraint must be removed.

What do you think?